bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java

```diff
@@ -589,6 +620,24 @@ public void onAnimationEnd(Animator animation) {
     }
 
     /**
+     * Controls the behavior on back button press when the state is {@link State#EXPANDED}.
+     *
+     * @param state {@link State#PEEKED} to show the peeked state on back press or {@link
+     *              State#HIDDEN} to completely hide the Bottom Sheet. Default is {@link  State#HIDDEN}.
+     */
+    public void setDismissMode(State state) {
```

Comment:
I agree that the boolean is better, and less confusing. Just to be clear, so... 

    /**
     * Controls the behavior on back button press when the state is {@link State#EXPANDED}.
     *
     * @param peekOnDismiss True if the peeked state should be shown on back press or false to completely hide sheet. Default is false.
     */
    setDismissMode(boolean peekOnDismiss)

keeping the same method name? Or should it just be something like

    setPeekOnDismiss(boolean)

    boolean isPeekOnDismiss()